images/ubuntu/scripts/build/install-azure-cli.sh

```diff
@@ -4,12 +4,40 @@
 ##  Desc:  Install Azure CLI (az)
 ################################################################################
 
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/etc-environment.sh
+
+# AZURE_CONFIG_DIR shell variable defines where the CLI configuration file for managing behavior are stored
+# https://learn.microsoft.com/en-us/cli/azure/azure-cli-configuration#cli-configuration-file
+# This path SHOULD be different from the installation directory /opt/az/
```

Comment:
The configuration directory of Azure CLI on an agent is `/home/runner/.azure`.

You may see this using this action:

```yaml
name: Run Azure Login

on:
  workflow_dispatch:

jobs:
  test:
    runs-on: ubuntu-24.04
    steps:   
    - uses: azure/login@v2
      with:
         creds: ${{ secrets.AZURE_CREDENTIALS }}

    - run: az account get-access-token --debug
```

The debug log shows:

```
DEBUG: cli.azure.cli.core.auth.persistence: build_persistence: location='/home/runner/.azure/service_principal_entries.json', encrypt=False
```